package.json

```diff
@@ -5,6 +5,7 @@
   "private": false,
   "homepage": "https://github.com/Esri/calcite-colors",
   "main": "colors.json",
```

Comment:
if you change where the colors json file is (I think you moved it to `dist`) you need to also update this line as it makes it the default require for npm